scripts/translate.js

```diff
@@ -101,7 +101,10 @@ diff.updated.forEach(key => console.info(`📥  [UPDATE] "${key}"`));
 // Save translations
 mkdirpSync(LANG_DIR);
 fs.writeFileSync(DEFAULT_TRANSLATIONS_FILE, convertToJSON(defaultMessages));
-translate('fr', defaultMessages, diff.updated);
 translate('es', defaultMessages, diff.updated);
+translate('fr', defaultMessages, diff.updated);
+translate('it', defaultMessages, diff.updated);
 translate('ja', defaultMessages, diff.updated);
+translate('pt', defaultMessages, diff.updated);
 translate('ru', defaultMessages, diff.updated);
+translate('zh', defaultMessages, diff.updated);
```

Comment:
@Betree any reason why we were skipping some languages here?

Reply:
I initially didn't want to add them before we have a good translation coverage for them, but adding them here is actually important to ensure the files stay up to date so I'm good with this change.